tox.ini

```diff
@@ -81,6 +81,7 @@ commands =
 setenv =
     DJANGO_SETTINGS_MODULE = test_settings
 deps =
+    -r{toxinidir}/requirements/base.txt
```

Comment:
Added Base deps here because without them Latest django(4.0) was being installed which caused issues regarding things dropped and deprecated in Django 4